server.py

```diff
@@ -8,8 +9,13 @@ def desk():
     return render_template("index.html")
 
 
+@app.route("/word/recognize/")

```

Comment:
Add also some parameter for the word. It should be something like this: `tagged/word/%string word%`.